course_catalog/models.py

```diff
@@ -18,3 +18,18 @@ class Course(models.Model):
     enrollment_end = models.DateTimeField(null=True)
     image = JSONField(null=True)
     raw_json = JSONField(null=True)
+
+
+class CourseInstructor(models.Model):
+    first_name = models.CharField(max_length=128)
+    last_name = models.CharField(max_length=128)
```

Comment:
Will every instructor have the `mit_id` field that could be added and used as a unique key?  Can `title` be added?

Reply:
`courses = models.ManyToManyField(Course)?`

Reply:
`title` seems difficult to ascertain apropos Ali's discussion in the PR description. What sort of values should we expect? The invision design only suggests "Prof." as a title.

Reply:
ok